LBBT-Design-Patterns/ITaxCalculator.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace LBBT_Design_Patterns
+{
+    internal interface ITaxCalculator
```

Comment:
Renamed as it seems as if LBBT is just one kind of tax so there wouldn't be multiple implementations of that, but there could be multiple different kinds of tax to calculate. That's just a design decision though.